app/components/content/content.js

```diff
@@ -299,7 +299,7 @@ const Content = ({ p2p, content, renderRow }) => {
               isLoading={isDeleting}
               onClick={async () => {
                 setIsDeleting(true)
-                await p2p.delete(content.rawJSON.url)
+                await p2p.delete(content.rawJSON.url, true)
```

Comment:
```suggestion
                const deleteFiles = true
                await p2p.delete(content.rawJSON.url, deleteFiles)
```

Unfortunately this API uses boolean arguments, which need some documentation